src/renderer/MainContent.tsx

```diff
@@ -30,6 +115,22 @@ const MainContent = () => {
               key={`slider-${sliderIndex}`}
             />
           ))}
+          <div className="col sliderButtons">
+            <LargeButton
+              ariaLabel=""
```

Comment:
can we add `ariaLabel`s here? It's good practice w.r.t. accessibility to have a label for input elements

Reply:
oops I missed this.